### DISCUSSION TITLE
Question: Where is the code to link android and ios dependencies while running prebuild ?

### DISCUSSION THREAD

**Discussion by vipulbhj** [zoontek/react-native-bootsplash#663] [General]
Hello, I am new to this codebase and not very familiar with this the Expo / React Native ecosystem, but I have beeb trying to read source code of famous projects and learn. I was browsing code for the project and found a ton of cool insights, but one thing I couldn't understand. 

How does the `RNBootSplash` become available on Android ? I understand some linking is happening, but how, I don't see any code for it and it feels like MAGIC ✨✨✨

Does anyone know ?

Would love to understand how that is happening 😄

**Comment by zoontek** [comment]
@vipulbhj [Read infos on autolinking here](https://github.com/react-native-community/cli/blob/main/docs/autolinking.md)
